README.md

```diff
@@ -1,10 +1,82 @@
 [![Build Status](https://travis-ci.org/kubernetes-csi/driver-registrar.svg?branch=master)](https://travis-ci.org/kubernetes-csi/driver-registrar)
 # Cluster Driver Registrar
 
-A sidecar container that
+The cluster-driver-registrar is a sidecar container that creates a cluster-level
+[CSIDriver object](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/csi-api/pkg/crd/manifests/csidriver.yaml)
+for the CSI driver.
+
+This sidecar container is only needed if you need one of the following Kubernetes
+features:
+
+* Skip attach: For drivers that don't support ControllerPublishVolume, this
+  eliminates the need to deploy the external-attacher sidecar
+* Pod info on mount: This passes Kubernetes metadata such as Pod name and
+  namespace to the NodePublish call
 
 1. Registers the containerized CSI driver with Kubernetes and create a CSIDriver object
 
+## Compatibility
+
+| Latest stable release                                                                                       | Branch                                                                                  | Compatible with CSI Version                                                                | Container Image                                 | Min K8s Version | Max K8s Version |
+| ----------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ | ----------------------------------------------- | --------------- | --------------- |
+| [cluster-driver-registrar v1.0.1](https://github.com/kubernetes-csi/cluster-driver-registrar/releases/tag/v1.0.1) | [release-1.0](https://github.com/kubernetes-csi/cluster-driver-registrar/tree/release-1.0) | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.1 | 1.13            | -               |
+
+## Usage
+
+### Features
```

Comment:
I would remove the features section here. Once https://github.com/kubernetes-csi/docs/pull/98/files is merged we can update the descriptions above (line 11 and 13) to link to the new features pages being created (`skip-attach.md` and `pod-info.md`).